python/tk_multi_loader/dialog.py

```diff
@@ -1481,6 +1512,40 @@ def _populate_entity_breadcrumbs(self):
 
         self.ui.entity_breadcrumbs.setText("<big>%s</big>" % breadcrumbs)
 
+    def _status_filter_change(self):
+        """
+        Reload the status filter combo box 
+        With correct count in brackets
+        """
+        tmp_index = self._filter_status.currentIndex()
+        tmp_index = 0 if tmp_index< 0 else tmp_index
+
+        publish_items = self._publish_model._publish_items
+        self._filter_status.clear()
+
+        status_count = {"All": 0}
+        valid_ids = self._publish_proxy_model._valid_type_ids
+
+        for item in publish_items:
+            if item['type_id'] in valid_ids:
+                stat = item['sg_item']['sg_status_list']
+                if stat in status_count:
+                    status_count[stat] += 1
+                else:
+                    status_count[stat] = 1
+                status_count["All"] += 1
+
+        self._filter_status.addItem("All ({0})".format(status_count['All']))
+        for status in STATUS_LIST:
+            icon = QtGui.QIcon()
+            if status in status_count:
+                name = "{0} ({1})".format(status, status_count[status])
+            else:
+                name = "{0} (0)".format(status)
+            icon.addPixmap(QtGui.QPixmap(":/res/{0}.png".format(status)), QtGui.QIcon.Normal, QtGui.QIcon.Off)
```

Comment:
This only works if you have added your icons to the resources.qrc file and compiled it in a py file.
Check the build_res function in resources/build_resources.sh.